core/templates/dev/head/filters.js

```diff
@@ -196,8 +196,28 @@ oppia.filter('parameterizeRuleDescription', [
           } else {
             // The following case is for MultipleChoiceInput
             for (var i = 0; i < choices.length; i++) {
+              var stringChange = true;
+              var choicesLabelString = choices[i].label;
+              while(stringChange) {
+                var nonInteractiveInputs = ['collapsible', 'image', 'link',
```

Comment:
Turn this into a constant, see [this example](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/filters.js#L19).